Firestore/tests/Unit/WriteBatchTest.php

```diff
@@ -348,24 +348,28 @@ public function testSentinelsInArray()
 
     public function testSentinelsAfterArray()
     {
-        $this->batch->set('name', [
+        $ret = $this->batch->set('name', [
             'foo' => [
                 'a', 'b', 'c'
             ],
             'bar' => FieldValue::serverTimestamp()
         ]);
+
+        $this->assertInstanceOf(\Google\Cloud\Firestore\BulkWriter::class, $ret);
```

Comment:
We recently deprecated `WriteBatch` in favour of `BulkWriter`, should this retain the old class name?
```suggestion
        $this->assertInstanceOf(\Google\Cloud\Firestore\WriteBatch::class, $ret);
```

Reply:
On second thought, maybe not coz the code has moved over to new class.

Reply:
It's a bit odd, since this IS the `WriteBatch` test. But since that class has been deprecated in favor of `BulkWriter`, I'd rather test the new class name.